README.md

```diff
@@ -76,8 +76,7 @@ After you're finished please remove all the comments and instructions!
 _ **[React & javascript functions, modules, ES6 syntax, variables, etc]**
 
 ## 🚀 Live Demo <a name="live-demo"></a>
-
-- Comming Soon
+- [Live](https://citec-47.github.io/MEALS-API/dist/)
```

Comment:
- [ ] I think it would be better if you update your `[Live Demo](https://citec-47.github.io/MEALS-API/dist/)
` link in your README file to make it more professional as it shows different project once clicked. If you need some helps of how to deploy a react project, I suggest you **[refer this article](https://www.youtube.com/watch?v=tVzpC5_AC8M)** or **from now you can remove the Live Demo and add it later once you have deployed your App**